lib/appdirs/__init__.py

```diff
@@ -0,0 +1 @@
+from .appdirs import user_cache_dir
```

Comment:
Normally a library like this would just be added as 'lib/appdirs.py' instead of in its own subfolder and adding an `init`.